### ISSUE TITLE
Demo doesn't work on Windows 8.1 / IE11

### ISSUE THREAD

**Issue by chrisdot** [fengyuanchen/cropper#328]
Hello,

I've tested your cropper control in various scenariis and really love it !!

But after an integration step, I started to test it through various touch enabled devices. So I switched back to your demo, and that confirmed that it is impossible to use it in the following configurations:
- on windows 8.1 / internet explorer 11 on my little atom based tablet (either desktop or metro style)
- on win8.1 on a regular desktop with ie11 (either desktop or metro style)

But it works on win7 / ie11 !!

On win8/IE11, it just looks like if the touch feature is disabled...

Cheers,
Christophe

**Comment by fengyuanchen** [comment]
Thank you for your feedback. I will try to check this while I get a window 8.1 system environment.

**Comment by pixxelfriend** [comment]
Hey, 

i tried it out yesterday on alot of devices, works great i think on Android and Windows Phone.
I know is that Windows Tablets are not working good with touch scripts, because they have their own pointer events. @fengyuanchen: To be sure, you should add them in the event list of the cropper.js : 

```javascript
 // Events
      EVENT_MOUSE_DOWN = 'mousedown touchstart pointerdown MSPointerDown',
      EVENT_MOUSE_MOVE = 'mousemove touchmove pointermove MSPointerMove',
      EVENT_MOUSE_UP = 'mouseup mouseleave touchend touchleave touchcancel pointerup pointercancel MSPointerUp MSPointerCancel',
```

You need to apply to the parent element of your image this css code: 
-ms-touch-action: none;touch-action: none;

This is my allround solution for every touch supported script on Windows Tablets and Phones.

Cheers, 
Steffen

**Comment by fengyuanchen** [comment]
@pixxelfriend Awesome! Thank you very much!

**Comment by fengyuanchen** [comment]
The new version with these events support had been released ([v0.10.1](https://github.com/fengyuanchen/cropper/releases/tag/v0.10.1)).
